CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [1.3.0] - 2021-03-24
+### Changes
+- Use inclusive language
+Use inclusive language in docs, variable names, and more
+
+### New features
+- Support ipv6.method disabled (8318686)
+- Support dns-options (880b7ab)
```

Comment:
Please explain which options and try to use more formal language instead of technical terms - also the actual setting would be with underscore and not with dashes. To me the commit IDs seem to be disturbing.

- Support to configure DNS options to ... and ...

Also noticed that the README does not explain which DNS options can be set and what they do and how they are written. Are these the same options that resolv.conf(5) describes? Then this could be written. If there are differences they should be explained.

Reply:
They are the same DNS options as resolv.conf(5) describing

Reply:
I will create a new PR describing the available DNS options

Reply:
But currently, we only support DNS options for **IPv4**

Reply:
DNS should not be specific to the address family. NetworkManager gets this wrong with `ipv4.dns-options` and `ipv6.dns-options`.

How to correctly abstract that in the role, is a different question.